### ISSUE TITLE
Colour contrast in textbox placeholders

### ISSUE THREAD

**Issue by james-rae** [ramp4-pcar4/highcharts-editor#57]
## Description

Interesting, as WAVE doesn't seem to care but my manual check failed.

This is for text like this:

![Image](https://github.com/user-attachments/assets/e9042bb6-785d-45b4-97e8-f332af9090b4)

Text appears to be `#AAB0BB`, and background is `#FFFFFF`.  Contrast is 2.17, which fails 🐋 .

[Internet says](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder#color_contrast) this kind of placeholder text still needs to respect the rules.
